google_compute_engine/accounts/accounts_daemon.py

```diff
@@ -37,20 +37,30 @@ class AccountsDaemon(object):
   invalid_users = set()
   user_ssh_keys = {}
 
-  def __init__(self, groups=None, remove=False, debug=False):
+  def __init__(
+      self, groups=None, remove=False,
+      useradd_cmd='useradd -m -s /bin/bash -p * {user}',
```

Comment:
One Python style nit - mind pulling these strings out into variables defined at the top of the file?